libsolutil/IpfsHash.cpp

```diff
@@ -53,36 +68,142 @@ string base58Encode(bytes const& _data)
 	reverse(output.begin(), output.end());
 	return output;
 }
+
+struct Chunk
+{
+	Chunk() = default;
+	Chunk(bytes _hash, size_t _size, size_t _blockSize):
+		hash(std::move(_hash)),
+		size(_size),
+		blockSize(_blockSize)
+	{}
+
+	bytes hash = {};
+	size_t size = 0;
+	size_t blockSize = 0;
+};
+
+struct Link
+{
+	Link() = default;
+	Link(size_t _size, size_t _blockSize, bytes _hash) : size(_size), blockSize(_blockSize), hash(std::move(_hash)) {}
```

Comment:
```suggestion
	Link(size_t _size, size_t _blockSize, bytes _hash): size(_size), blockSize(_blockSize), hash(std::move(_hash)) {}
```

Reply:
I removed Link structure, so this is also fixed